build/target-repository/docs/rector_rules_overview.md

```diff
@@ -110,7 +110,7 @@ use Rector\Arguments\Rector\ClassMethod\ArgumentAdderRector;
 use Rector\Arguments\ValueObject\ArgumentAdder;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
-return static function (\Rector\Config\RectorConfig $containerConfigurator): void {
+return static function (ContainerConfigurator $containerConfigurator): void {
```

Comment:
@TomasVotruba run `composer docs` seems back to `ContainerConfigurator` usage instead of `Rector\Config\RectorConfig`, it seems `vendor/bin/rule-doc-generator` need some flag to use specific configurator class.

Reply:
Yeah, the symplify/rule-doc-generator will need an update for this one